### PR TITLE
show movement for all entity types on unit tool tip

### PR DESCRIPTION
**Pull request by kuronekochomusuke**
- show movement mode for all entity types on unit tool tip.  to make it easier to see what mode unit is in.  most useful for units that can convert modes.
- correct issue of movement reduced bomb indicator incorrectly showing on LAMs.

image

image

### CODE REVIEW

File: megamek/src/megamek/common/LandAirMech.java [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ public void clearBombChoices() {
     @Override
     public int reduceMPByBombLoad(int t) {
         // bombs don't impact movement
-        return 0;
```
Comment (Windchild292) [inline megamek/src/megamek/common/LandAirMech.java:1133]: That goes against the comment, and doesn't appear related?

**Review by Windchild292 — CHANGES_REQUESTED** [review]
It looks like there's an unintended change included in this.

**Comment by Windchild292** [comment]
I'm just blind apparently, but does that cause issues with the actual movement given the very obvious comment above the changed LAM line.

File: megamek/src/megamek/common/LandAirMech.java (hunk 1 above)
Comment (kuronekochomusuke) [reply megamek/src/megamek/common/LandAirMech.java:1133]: the movement it not impacted so, so it should return the same value.

**Review by Windchild292 — APPROVED** [review]
